cypress/e2e/awx/resources/inventoriesConstructed.cy.ts

```diff
@@ -64,28 +59,30 @@ describe('Constructed Inventories CRUD Tests', () => {
     cy.clickButton(/^Create constructed inventory$/);
     cy.getByDataCy('name').type(constInvName);
     cy.getByDataCy('description').type(`Description of "${constInvName}" typed by Cypress`);
-    cy.intercept({
-      method: 'GET',
-      pathname: awxAPI`/organizations/`,
-      query: { name__icontains: organization.name },
-    }).as('filterOrg');
     cy.singleSelectBy('[data-cy="organization"]', organization.name);
-    cy.wait('@filterOrg');
     // this can be simplified if we include data-cy to the search button of instance groups
     cy.multiSelectByDataCy('instance-group-select-form-group', [instanceGroup.name]);
 
-    cy.intercept({
-      method: 'GET',
-      pathname: awxAPI`/instance_groups/`,
-      query: { name: instanceGroup.name },
-    }).as('filterInstanceG');
-    cy.intercept({
-      method: 'GET',
-      pathname: awxAPI`/inventories/`,
-      query: { name__icontains: invNames[invToCreate - 1] },
-    }).as('filterInputInventories');
-    cy.multiSelectBy('[data-cy="inventories"]', invNames);
-    cy.wait('@filterInputInventories');
+    //get inventories select
+    cy.get('form')
+      .contains('button', /^Select inventories$/)
+      .click();
+    cy.get('form')
+      .document()
+      .its('body')
+      .find('.pf-v5-c-menu__content')
+      .should('have.length', 1)
+      .within(() => {
+        cy.getByDataCy('search-input').within(() => {
+          cy.get('input').clear().type(inventory.name);
+        });
+        cy.contains('.pf-v5-c-menu__item-text', inventory.name)
+          .parent()
+          .within(() => {
+            cy.get('input').click();
+          });
+      });
+
```

Comment:
```suggestion
    //get inventories select
  cy.multiSelectBy('input-inventories', [inventory.name])
```
I think that'll work.

Reply:
I'd definitely prefer to use the custom command, but it seems to be a source of flakiness for this test/field. The changes I've made run consistently and use the same code as the singleSelectBy, just searching instead for a button with the correct text, and adding an assertion that there should only be one result.

Reply:
Met with Alex and decided to implement ```cy.multiSelectByDataCy()```